ansible_collections/arista/avd/molecule/cv_deploy/converge.yml

```
@@ -158,18 +157,18 @@
         cv_submit_workspace_force: true
         cv_run_change_control: true
 
-    - name: Test = Provision with cv_submit_workspace_force = false
+    - name: Provision with cv_submit_workspace_force = false
```

Stop streaming on core1 and wait to 130s seconds for streaming status turbine to notice (may need up to 2 minutes)
```suggestion
    - name: Force device avd-ci-core1 to stop streaming
      arista.eos.eos_command:
        commands:
          - enable
          - configure
          - daemon TerminAttr
          - shutdown
      when: inventory_hostname == "avd-ci-core1"
    
    - name: Pause for 130 seconds for streaming to stop
      run_once: true
      ansible.builtin.pause:
        seconds: 130

    - name: Provision with cv_submit_workspace_force = false
```